tests/phpunit/test-admin-interface.php

```diff
@@ -125,4 +125,101 @@ public function test_delete_transient_on_option_update() {
 		$this->assertFalse( $cached_max_age_compare );
 	}
 
+	/**
+	 * Test setting the max age to the default value if it was set to 600.
+	 */
+	public function test_set_max_age_to_default() {
```

Comment:
Once again will raise that these should be a series of tests for the same method, not a single test with assertions mixed in.

Reply:
added messages to clarify the checks on failure